frontend/src/metabase/admin/datamodel/components/FieldRemapping.jsx

```diff
@@ -254,52 +255,55 @@ export default class FieldRemapping extends React.Component {
 
     return (
       <div>
-        <Select
-          value={mappingType}
-          onChange={this.handleChangeMappingType}
-          options={this.getAvailableMappingTypes()}
-          optionValueFn={o => o}
-        />
-        {mappingType === MAP_OPTIONS.foreign && [
-          <SelectSeparator key="foreignKeySeparator" />,
-          <PopoverWithTrigger
-            ref="fkPopover"
-            triggerElement={
-              <SelectButton
-                hasValue={hasFKMappingValue}
-                className={cx("flex inline-block no-decoration", {
-                  "border-error": dismissedInitialFkTargetPopover,
-                  "border-dark": !dismissedInitialFkTargetPopover,
-                })}
-              >
-                {fkMappingField ? (
-                  fkMappingField.display_name
-                ) : (
-                  <span className="text-medium">{t`Choose a field`}</span>
-                )}
-              </SelectButton>
-            }
-            isInitiallyOpen={isChoosingInitialFkTarget}
-            onClose={this.onFkPopoverDismiss}
-          >
-            <FieldList
-              className="text-purple"
-              field={fkMappingField}
-              fieldOptions={{
-                count: 0,
-                dimensions: [],
-                fks: this.getForeignKeys(),
-              }}
-              table={table}
-              onFieldChange={this.onForeignKeyFieldChange}
-              hideSingleSectionTitle
-            />
-          </PopoverWithTrigger>,
-          dismissedInitialFkTargetPopover && (
-            <div className="text-error my2">{t`Please select a column to use for display.`}</div>
-          ),
-          hasChanged && hasFKMappingValue && <RemappingNamingTip />,
-        ]}
+        <Flex className="align-center">
```

Comment:
```suggestion
        <Flex align="center">
```

You can actually just use a specific align prop here instead of className. They result in the same thing but I find that a little cleaner.